app/migrations.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS migrations (
+    last_migration_date TIMESTAMP
+);
+
+TRUNCATE TABLE migrations;
+
+INSERT INTO migrations (last_migration_date)
+VALUES (NOW());
+
+SELECT * FROM migrations;
```

Comment:
Not sure if this is consistent with the style on this project, but my nit suggestion would be adding a comment to this (and the db-migrate sample script above) to indicate these are examples and not real functionality -- for a second I thought we were implementing our own migration handler from scratch 😅

Reply:
https://github.com/navapbc/template-infra/pull/332